language-extensions/R/test/src/RExtensionApiTests.cpp

```diff
@@ -32,12 +32,13 @@
 #include "Common.h"
 
 using namespace std;
-namespace fs = experimental::filesystem;
 
 #ifdef _WIN64
 	const string x_RExtensionLibName = "libRExtension.dll";
+	namespace fs = filesystem;
 #else
 	const string x_RExtensionLibName = "libRExtension.so.1.1";
+	namespace fs = experimental::filesystem;
```

Comment:
Is it still experimental on linux?

Reply:
For the docker image (already the updated one) we use for build pipeline, it is still experimental.